scripts/prolog

```diff
@@ -7,60 +7,77 @@ This site contains Haskell documentation of
-              * __[LocalTxMonitor](ouroboros-network/Ouroboros-Network-Protocol-LocalTxMonitor-Type.html)__
-              * __[Handshake](ouroboros-network-framework/Ouroboros-Network-Protocol-Handshake-Type.html)__
-          
-         * /Block chain related data structures that the networking code needs
-           to be aware of:/
+              * __[inbound](ouroboros-network/Ouroboros-Network-TxSubmission-Inbound.html)__
+              * __[outbound](ouroboros-network/Ouroboros-Network-TxSubmission-Outbound.html)__
 
-              * __[Block](ouroboros-network/Ouroboros-Network-Block.html)__
-              * __[AnchoredFragment](ouroboros-network/Ouroboros-Network-AnchoredFragment.html)__
+          * __[Peer Selection](ouroboros-network/Ouroboros-Network-PeerSelection-Governor.html)__
+          * __[Data Diffusion](ouroboros-network/Ouroboros-Network-Diffusion.html)__ which combines all the pieces for supporting
+            a [node](https://github.com://input-output-hk/cardano-node)
+          * __[NodeToNode API](ouroboros-network/Ouroboros-Network-NodeToNode.html)__
+          * __[NodeToClient API](ouroboros-network/Ouroboros-Network-NodeToClient.html)__
 
-         * /Network applications:/
+== Ouroboros Consensus
 
-              * __[Block Fetch Application](ouroboros-network/Ouroboros-Network-BlockFetch.html)__
-              * __Tx Submission Applications__:
+This graph depicts the relationship between the various `xxx-consensus-yyy` packages:
 
-                  * __[inbound](ouroboros-network/Ouroboros-Network-TxSubmission-Inbound.html)__
-                  * __[outbound](ouroboros-network/Ouroboros-Network-TxSubmission-Outbound.html)__
+![Consensus packages dependencies](packages-consensus.svg)
 
-              * __[Peer Selection](ouroboros-network/Ouroboros-Network-PeerSelection-Governor.html)__
-              * __[Data Diffusion](ouroboros-network/Ouroboros-Network-Diffusion.html)__ which combines all the pieces for supporting
-                a [node](https://github.com://input-output-hk/cardano-node)
-              * __[NodeToNode API](ouroboros-network/Ouroboros-Network-NodeToNode.html)__
-              * __[NodeToClient API](ouroboros-network/Ouroboros-Network-NodeToClient.html)__
-      
+* __ouroboros-consensus__ : Main package for Cardano consensus implementation, aka. Ouroboros.
+
+    * The [Ouroboros Consensus Modules Map](./modules-consensus.svg) represents the dependencies between various /Consensus/ modules and is a map to navigate the (large) codebase. Clicking on a module should bring you to the corresponding haddock page from which you can later navigate to the source code. __Note__: The image needs to be enlarged to be useful hence why it's an external link.
+
+* __ouroboros-consensus-test__ : Property and model-based tests for consensus, using [io-sim](https://)
+
+    * __[Network Simulator](ouroboros-consensus-test/Test-ThreadNet-Network.html)__: Simulates a network of nodes
```

Comment:
```suggestion
    * __[ThreadNet Simulator](ouroboros-consensus-test/Test-ThreadNet-Network.html)__: Simulates a network of nodes
```

There's another thing which we refer to as a [network simulator](https://input-output-hk.github.io/ouroboros-network/ouroboros-network-framework/Simulation-Network-Snocket.html#v:withSnocket).